### DISCUSSION TITLE
CB - how to count calls as slow with usage of ignored-exceptions

### DISCUSSION THREAD

**Discussion by podmak** [resilience4j/resilience4j#2136] [Q&A] [answered]
hi, 
I have set property `ignore-exceptions` in circuitbreaker configuration to define my business exception which I want to ignore from counting as fail call, but I want to still marked this call as slow, if `slow-call-duration-threshold` is exceeded.

Is it possible to configure `ignore-exceptions` to be ignored only for fail call but counted for slow calls?

thanks

**Comment by RobWin** [comment, marked as answer]
No, that's not possible :( 
That would be a new feature which needs to be implemented

**Reply by podmak** [reply]
thanks for quick response.
are you willing/able to add this as new feature somewhere to your roadmap?

**Reply by podmak** [reply]
hi @RobWin, what's your opinion about this?

**Reply by RobWin** [reply]
I won't be able to implement this.
But we are open for PR and contributions.
